packages/viewer/src/lib/types/interactions.ts

```diff
@@ -18,10 +18,16 @@ export interface InteractionConfig {
    * update the camera.
    */
   interactionDelay: number;
+
+  /**
+   * The amount of time between two clicks to be considered a double click.
+   */
+  doubleClickDelay: number;
 }
 
 export const defaultInteractionConfig: InteractionConfig = {
   finePointerThreshold: 1,
   coarsePointerThreshold: 3,
   interactionDelay: 75,
+  doubleClickDelay: 250,
```

Comment:
I took this number from the current implementation in manufacturing-ui. The _interactionDelay_ listed above was too short for the the double click to be triggered.